tutorials/pdes/heat-1-an-introduction.md

```diff
@@ -0,0 +1,179 @@
+# The Heat Equation
+
+**Authors**: Manuel Madeira, David Carvalho
+
+**Reviewers**: Fábio Cruz
+
+
+In this tutorial, we delve into the problem of ... 🔥 **heat diffusion** on a plate! 🔥
```

Comment:
Maybe you want to move a bit this text (or a sunmarized version of it) to the top page, as intro to the tutorial, and then have heat-1.md maybe repeat it again, with more explicit information about the structure (as it is). Maybe you also want to choose an image to put on the cover page of the tutorials.

Reply:
thanks, done.